Update README with updating instructions

DiegoPino opened the pull request:
# WHAT IS NEW

Added some basic instructions on updating a running archipelago build from the `minio` branch Experience can vary depending on how much customization has been done but then also, we can assume the user understands some basics of composer and git and could require less detailed instructions.

Some proofreading could help me here.

Thanks

DiegoPino committed:
Update README with updating instructions

Added some basic instructions on updating a running archipelago. Experience can vary depending on how much customization has been done but then also, means the user understands some basics of composer and git and

giancarlobi commented on README.md:
````diff
@@ -49,11 +55,72 @@ This will give you an admin user with archipelago as password (!change this if r
+That last command will use the updated `composer.lock` file to download/use exactly the same package versions and dependencies we have tested during development. Just in case and to see Archipelago is still in shape, clear your Drupal caches
+
+```Shell
+docker exec -ti esmero-web bash -c "cd web; drush cr"
````
@DiegoPino I used 
`docker exec -ti esmero-web bash -c "drush cr"` 
without `cd web`
I don't know if that depend on my installation, just in case.

DiegoPino replied:
Yes! you are right.
I guess it depends on the installation really and if drush was deployed globally?  I had some issues in the past calling drush outside the folder of the installation.  to be sure i just tested both with -d option

```SHELL
[preflight] Config paths: /var/www/html/vendor/drush/drush/drush.yml,/var/www/html/drush/drush.yml
 [preflight] Alias paths: /var/www/html/web/drush/sites,/var/www/html/drush/sites
 [preflight] Commandfile search paths: /var/www/html/vendor/drush/drush/src,/var/www/html/drush
 [bootstrap] Starting bootstrap to site [0.73 sec, 3.22 MB]
 [bootstrap] Drush bootstrap phase 2 [0.73 sec, 3.22 MB]
 [bootstrap] Try to validate bootstrap phase 2 [0.74 sec, 3.22 MB]
 [bootstrap] Try to validate bootstrap phase 2 [0.74 sec, 3.22 MB]
 [bootstrap] Try to bootstrap at phase 2 [0.76 sec, 3.29 MB]
 [bootstrap] Drush bootstrap phase: bootstrapDrupalRoot() [0.76 sec, 3.29 MB]
 [bootstrap] Change working directory to /var/www/html/web [0.76 sec, 3.29 MB]
 [bootstrap] Initialized Drupal 8.6.14 root directory at /var/www/html/web [0.76 sec, 3.29 MB]
 [bootstrap] Try to validate bootstrap phase 2 [0.76 sec, 3.29 MB]
 [bootstrap] Try to bootstrap at phase 2 [0.8 sec, 3.39 MB]
 [bootstrap] Drush bootstrap phase: bootstrapDrupalSite() [0.8 sec, 3.39 MB]
 [bootstrap] Initialized Drupal site default at sites/default [0.8 sec, 3.39 MB]
 [success] Cache rebuild complete. [22.14 sec, 91.55 MB]
````

Same output!

Will fix, thanks!

DiegoPino committed:
proofreading fixes

in our docker "drush cr" works the same way if invoked from /web or the html root. I checked twice! Thanks @giancarlobi i learn new things everyday =)

giancarlobi reviewed (approved):
Great.
